api/api/src/test/scala/uk/ac/wellcome/platform/api/works/WorksTestInvisible.scala

```diff
@@ -38,7 +38,7 @@ class WorksTestInvisible extends ApiWorksTestBase {
     withWorksApi {
       case (worksIndex, routes) =>
         val work = indexedWork().title("This shouldn't be deleted!")
-        insertIntoElasticsearch(worksIndex, work, deletedWork)
+        insertIntoElasticsearch(worksIndex, work, invisibleWork)
```

Comment:
I am also wondering if this test is dubious, for similar reasons.

Reply:
It is, will fix